admin/database/postgres/postgres.go

```diff
@@ -230,6 +230,18 @@ func (c *connection) DeleteOrganizationWhitelistedDomain(ctx context.Context, id
 	return checkDeleteRow("org whitelist domain", res, err)
 }
 
+func (c *connection) FindInactiveOrganizations(ctx context.Context) ([]*database.Organization, error) {
+	// TODO: This definition may change, but for now, we are considering an organization as inactive if it has no users
+	res := []*database.Organization{}
+	err := c.getDB(ctx).SelectContext(ctx, &res, `
+		SELECT o.* FROM orgs o WHERE NOT EXISTS ( SELECT 1 FROM users_orgs_roles uor WHERE uor.org_id = o.id )
```

Comment:
Not perfect, but maybe throw in something like a `now() - updated_on > INTERVAL '1 DAY'` to prevent too eager cleanup if some future refactors cause orgs to not have a member for a short while